### PR TITLE
Model golint

### LINKED ISSUE #90: Address golint recommendations in package model
Currently golint has 730 recommendations for package model. Go over these, evaluate and address the ones that we agree with.

Many of these are easy, e.g. missing comments on some exported functions and can be addressed head on.  Some of those recommendations may need to await next major upgrade.

Thus identify recommendations that we want to address in next major version and either list them in this ticket or create a new ticket with a label (v3).

### PR DESCRIPTION
**Pull request by adrg**
Address golint notices in the model package.
Resolves #90

### CODE REVIEW

File: pdf/model/writer.go [hunk 1]
```diff
@@ -676,17 +676,17 @@ type EncryptOptions struct {
...
 
 const (
-	// RC4_128bit uses RC4 encryption (128 bit)
-	RC4_128bit = EncryptionAlgorithm(iota)
-	// AES_128bit uses AES encryption (128 bit, PDF 1.6)
-	AES_128bit
-	// AES_256bit uses AES encryption (256 bit, PDF 2.0)
-	AES_256bit
+	// RC4128bit uses RC4 encryption (128 bit)
+	RC4128bit = EncryptionAlgorithm(iota)
```
Comment (gunnsth) [inline pdf/model/writer.go]: RC4128bit is confusing. RC4_128bit clearly better.  Would keep the others same style for consistency here.

File: pdf/model/colorspace.go [hunk 2]
```diff
@@ -2722,13 +2835,15 @@ func (cs *PdfColorspaceDeviceN) ColorFromPdfObjects(objects []core.PdfObject) (P
 	return cs.ColorFromFloats(floats)
 }
 
+// ColorToRGB converts a DeviceN color to a RGB color.
```
Comment (gunnsth) [inline pdf/model/colorspace.go]: Should be "an RGB color"

File: pdf/model/colorspace.go [hunk 3]
```diff
@@ -39,9 +39,9 @@ import (
 type PdfColorspace interface {
 	// String returns the PdfColorspace's name.
 	String() string
-	// ImageToRGB converts an Image in a given PdfColorspace to an RGB image.
+	// ImageToRGB converts an Image in a given PdfColorspace to a RGB image.
```
Comment (gunnsth) [inline pdf/model/colorspace.go]: "an RGB image"

File: pdf/model/colorspace.go [hunk 4]
```diff
@@ -1893,6 +1986,7 @@ func (cs *PdfColorspaceICCBased) ColorFromPdfObjects(objects []core.PdfObject) (
 	return cs.Alternate.ColorFromPdfObjects(objects)
 }
 
+// ColorToRGB converts a ICCBased color to a RGB color.
 func (cs *PdfColorspaceICCBased) ColorToRGB(color PdfColor) (PdfColor, error) {
 	/*
```
Comment (gunnsth) [inline pdf/model/colorspace.go]: Remove commented code block

**Comment by adrg** [comment]
@gunnsth I'm going to close this pull request and make another to resolve the history issue properly.

**Pull request closed without merging**